### PR TITLE
Add Stellaswap Pools

### PR DESCRIPTION
**Pull request by cartman-dev**
* Add Acala and Interlay tokens xcACA, xciBTC, xcINTR
* Add Stellaswap pools stellaswap-wglmr-xcibtc, stellaswap-wglmr-xcausd, stellaswap-wglmr-xcintr
* Add dummy xcACA pool for price resolution - Stella xcAUSD-WGLMR rewards with xcACA

### CODE REVIEW

File: src/data/moonbeam/stellaswapLpPools.json [hunk 1]
```diff
@@ -1,4 +1,23 @@
 [
+  {
+    "name": "stellaswap-xcaca-wglmr",
+    "address": "0xb83c50eA82c10AC598BA1ffc0c0296978c29c1e2",
+    "decimals": "1e18",
+    "poolId": 0,
```
Comment (MirthFutures) [inline src/data/moonbeam/stellaswapLpPools.json:6]: This doesn't seem like the correct pid?

**Review by MirthFutures — COMMENTED** [review]
Can you verify the pid is correct?

File: src/data/moonbeam/stellaswapLpPools.json (hunk 1 above)
Comment (MirthFutures) [reply src/data/moonbeam/stellaswapLpPools.json:6]: Nevermind I see you made it a dummy pool. This should actually go into a different folder. https://github.com/beefyfinance/beefy-api/blob/master/src/data/archive/oldLpPools.json

**Comment by EPETE-EPETE** [comment]
looks like pid 20, and should be xcausd instead of xcaca (xcaca is one of the reward tokens)
https://moonbeam.moonscan.io/tx/0x5ad1adc59658db6efb9f43704e6409087e40069f3ed85b7effd2ec28574043d1

File: src/data/moonbeam/stellaswapLpPools.json (hunk 1 above)
Comment (cartman-dev) [reply src/data/moonbeam/stellaswapLpPools.json:6]: Got it, I will move it to oldLpPools.json

**Pull request merged**